### PR TITLE
Update to Added support for HeliOS 10Ghz failover to 5Ghz state sensor

### PR DESCRIPTION
**Pull request by markoh76**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/discovery/helios.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    sensors:
+        state:
+            data:
+                -
+                    oid: IGNITENET-MIB::mlRadioStatusFailoverStatus.2
+                    value: mlRadioStatusFailoverStatus
+                    num_oid: .1.3.6.1.4.1.47307.1.4.3.1.2.2
+                    descr: 60GHz to 5GHz Radio Failover Status
+                    states:
+                        - { value : 0, generic : 0, graph : 0, descr : Failover Inactive }
```
Comment (murrant) [inline includes/definitions/discovery/helios.yaml]: Maybe better descriptions would be:
"Failover Status"
"60GHz"
"5GHz"

I think that would be a lot cleaner.  And you get the yellow color to indiate 5GHz is the failover status.

**Comment by murrant** [comment]
You will probably need to update the ceraos test data too.

Running ./scripts/collect-snmp-data.php -o ceraos will update the snmprec file with data for the new oids you added and ./scripts/save-test-data.php -o ceraos will update the json db dump.

**Comment by murrant** [comment]
@markoh76 looks like you have a different author for some of your commits, you can either squash them all or: https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit

**Comment by laf** [comment]
@markoh76 Any update on this?

**Comment by laf** [comment]
Please feel free to respond to this and we can re-open.

**Pull request closed without merging**